src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -50,6 +45,11 @@ class Feedback extends React.Component {
     return skillObj;
   };
 
+  state = {
+    feedbackInProgress: false,
+    skill: this.parseSkill(),
```

Comment:
earlier parseSkill() was called before being declared

Reply:
It will work but I don't think it's a good way of declaring state after any function as its a basic structure for components to declare state above all. We can look for some other option in this case.